remme/shared/forms/node_account.py

```diff
@@ -0,0 +1,13 @@
+from wtforms import fields, validators
+
+from .base import ProtoForm
+
+
+class NodeAccountInternalTransferPayloadForm(ProtoForm):
+    value = fields.IntegerField(validators=[
+        validators.DataRequired(message='Could not transfer with zero amount.')
+    ])
+
+
+class EmptyPayloadForm(ProtoForm):
+    pass
```

Comment:
All files should contain blank line at the end.

References:
- https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-file

Reply:
Done.